### PR TITLE
add db sync test case for test_ts_col_drop test_ts_col_alter_type test_ts_col_order_by

### PR DESCRIPTION
**Pull request by wyxxxcat**
issue : https://github.com/apache/doris/issues/43560

### CODE REVIEW

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by w41ter** [comment]
Please place cases in separate directories, so that we can run those cases in parallel. @wyxxxcat

File: regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    def values = [];
+    for (int index = 0; index < insert_num; index++) {
+        values.add("(${test_num}, ${index}, ${index})")
+    }
+    sql """
+        INSERT INTO ${tableName} VALUES ${values.join(",")}
+        """
+    sql "sync"
+
+    helper.ccrJobCreate()
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy:77]:
```suggestion
	helper.ccrJobDelete()
    helper.ccrJobCreate()
```

Comment (w41ter) [reply regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy:77]: We will run a case twice.

File: regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    sql "sync"
+
+    logger.info("=== Test 2: Check column type ===")
+
+    assertTrue(helper.checkShowTimesOf("""
+                                SHOW ALTER TABLE COLUMN
+                                FROM ${dbName}
+                                WHERE TableName = "${tableName}" AND State = "FINISHED"
+                                """,
+                                has_count(1), 30))
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy]: It should be has_count(2) ?

File: regression-test/suites/db_sync/column/drop/test_ds_col_drop_key_col.groovy [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    def values = [];
+    for (int index = 0; index < insert_num; index++) {
+        values.add("(${test_num}, ${index}, ${index})")
+    }
+    sql """
+        INSERT INTO ${tableName} VALUES ${values.join(",")}
+        """
+    sql "sync"
+
+    helper.ccrJobCreate()
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/drop/test_ds_col_drop_key_col.groovy]:
```suggestion
	helper.ccrJobDelete()
    helper.ccrJobCreate()
```

File: regression-test/suites/db_sync/column/drop/test_ds_col_drop_val_col.groovy [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    def values = [];
+    for (int index = 0; index < insert_num; index++) {
+        values.add("(${test_num}, ${index}, ${index})")
+    }
+    sql """
+        INSERT INTO ${tableName} VALUES ${values.join(",")}
+        """
+    sql "sync"
+
+    helper.ccrJobCreate()
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/drop/test_ds_col_drop_val_col.groovy]:
```suggestion
	helper.ccrJobDelete()
    helper.ccrJobCreate()
```

File: regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        // Field == 'value' && 'Type' == 'bigint'
+        return res[2][0] == 'value' && res[2][1] == 'bigint'
+    }
+
+    def id_is_big_int = { res -> Boolean
+        // Field == 'id' && 'Type' == 'bigint'
+        return res[1][0] == 'id' && res[1][1] == 'bigint'
+    }
+
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableName}"
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy:48]:
```suggestion
	helper.enableDbBinlog()
    sql "DROP TABLE IF EXISTS ${dbName}.${tableName}"
```

Comment (w41ter) [reply regression-test/suites/db_sync/column/alter_type/test_ds_col_alter_type.groovy:48]: ditto, for all cases

File: regression-test/suites/db_sync/column/order_by/test_ds_col_order_by.groovy [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    def tableName = "tbl_" + helper.randomSuffix()
+    def test_num = 0
+    def insert_num = 5
+
+    def exist = { res -> Boolean
+        return res.size() != 0
+    }
+
+    def key_columns_order = { res -> Boolean
+        // Field == 'id' && 'Key' == 'YES'
```
Comment (w41ter) [inline regression-test/suites/db_sync/column/order_by/test_ds_col_order_by.groovy]:
```suggestion
```

**Review by w41ter — CHANGES_REQUESTED** [review]

**Review by w41ter — APPROVED** [review]

**Pull request merged**